internal/pkg/dependencies/git_test.go

```diff
@@ -19,6 +20,10 @@ import (
 func TestGitRepositoryFs_SparseCheckout(t *testing.T) {
 	t.Parallel()
 
+	if runtime.GOOS == "windows" {
+		t.Skip("unstable on windows - random timeouts")
+	}
+
```

Comment:
V tomto teste to uz vzdavam, je tam retry, ale prikaz ani po 30 sekundach opakovane nedobehne s nejakou rozumnou hlaskou.